schema/source.schema.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/WICG/attribution-reporting-api/source.schema.json",
+  "title": "Source",
+  "description": "An Attribution Reporting source registration",
+  "type": "object",
+  "properties": {
+    "aggregatable_report_window": {
+      "$ref": "#/$defs/signed_duration_in_seconds"
+    },
+    "aggregation_keys": {
+      "type": "object",
+      "additionalProperties": {
```

Comment:
It's currently not in the spec, but shall we add the bytes limit in the schema? Similar for filters.

Reply:
It's not in the spec right now, so no.